new/LIT7026PrayerGod.xml

```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7026PrayerGod" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Prayer to God the Father</title>
```

Comment:
`en` (the language of the title used)

Reply:
(Or t1 gez ኦእግዚኦ፡ ነአኵተከ፡ ወንሴብሐከ፡ ወንሰግድ፡ ለዕበየ፡...)

Reply:
I think this is not updated

Reply:
Thank you!